roddy.sh

```diff
@@ -4,9 +4,11 @@ cd `dirname $0`
 RODDY_DIRECTORY=`readlink -f .`
 parm1=${1-}
 projectAnalysisParameter=$2
-export GROOVY_BINARY=groovy
-#export GROOVY_BINARY=/data/michael/Projekte/Roddy/groovyserv-1.1.0/bin/groovyclient
 
+#/data/michael/bin/groovyserv-1.1.0/bin/groovyserver -p 20000 & serverpid=$!
+export GROOVY_BINARY=groovy
+export GROOVY_BINARY="/data/michael/bin/groovyserv-1.1.0/bin/groovyclient -p 20000"
```

Comment:
I'll take care of it but it might already be done in  #81, leave it here and correct it. I also have to take care of multi-user environments and GroovyServ. The current implementation does not cover it!